packages/web-forms/src/components/common/map/createFeatureCollectionAndProps.ts

```diff
@@ -48,11 +48,11 @@ const getGeoJSONCoordinates = (geometry: string) => {
 
 const getGeoJSONGeometry = (coords: Coordinates[]): Geometry => {
 	if (coords.length === 1) {
-		return { type: 'Point', coordinates: coords[0] };
+		return { type: 'Point', coordinates: coords[0]! };
```

Comment:
These here and below have already been validated as not being undefined by `getGeoJSONCoordinates` and `createFeatureCollectionAndProps`. However, after the upgrade, it couldn't infer the type.